llvm/lib/Target/AMDGPU/GCNRegPressure.h

```diff
@@ -128,14 +128,7 @@ class GCNRPTracker {
 
   void clearMaxPressure() { MaxPressure.clear(); }
 
-  GCNRegPressure getPressure() const { return CurPressure; }
-
-  // returns MaxPressure, resetting it
-  decltype(MaxPressure) moveMaxPressure() {
-    auto Res = MaxPressure;
-    MaxPressure.clear();
-    return Res;
-  }
+  const GCNRegPressure &getPressure() const { return CurPressure; }
```

Comment:
Was this change intentional? Returning a ref seems less safe, because the caller has to be careful not to use it after mutating the tracker in any way.

Reply:
Done.